cmd/ctl/pkg/status/certificate/certificate.go

```diff
@@ -162,14 +164,27 @@ func (o *Options) Run(args []string) error {
 	util.DescribeEvents(crtEvents, prefixWriter, 0)
 	tabWriter.Flush()
 
-	issuerFormat := `Issuer:
-  Name: %s
-  Kind: %s`
 	issuerKind := crt.Spec.IssuerRef.Kind
 	if issuerKind == "" {
 		issuerKind = "Issuer"
 	}
-	fmt.Fprintf(o.Out, fmt.Sprintf(issuerFormat+"\n", crt.Spec.IssuerRef.Name, issuerKind))
+	// Get info on Issuer/ClusterIssuer
+	if issuerKind == "Issuer" {
```

Comment:
We would want to take in account that an issuer can also be from the not cert-manager group but still be called an issuer e.g. `cert-manager.primekey.com/Issuer`